### PR TITLE
Set up testing environment for Client

### PR DESCRIPTION
**Pull request by kaylawoodbury**
# [PT Story](https://www.pivotaltracker.com/story/show/171698315)
## Changes proposed in this pull request:
* set up Cypress and Enzyme/Jest for testing
* set up base url on cypress json file
* removed example files from cypress/integration
* set up script for "cy:open": "yarn start & cypress open" on package.json file
## Packages/Gems added
- enzyme: ^3.11.0
- enzyme-adapter-react-16: ^1.15.2
- cypress: ^4.1.0

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "baseURL": "http://localhost:3000"
```
Comment (tochman) [inline cypress.json]: This should probably be 3001 since you will be working with a Rails based API

File: src/setupTests.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { configure } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+
+configure ({ adapter: new Adapter() })
```
Comment (tochman) [inline src/setupTests.js]: not sure if important but I would skip the space between `configure` and the parenthesis

File: cypress.json (hunk 1 above)
Comment (kaylawoodbury) [reply cypress.json]: Updated to 3001

File: src/setupTests.js (hunk 2 above)
Comment (kaylawoodbury) [reply src/setupTests.js]: Always good to follow to follow best practices with formatting, thanks for pointing it out!  I've removed the extra space now

**Pull request merged**